File: C Modules/Assignments/Assignment no 6/Question-1/Type_1/Assignment no 1/Question_3.c
#include<stdio.h>
void leap();
int main()
{
    leap();
    return 0; 
}
void leap(){
    int year=2008;
    if ((year%400==0)||(year%4==0&&year%100!=0))
    {
        printf("%d is a leap year",year);
    }
    else
    {
        printf("%d is not a leap year",year);
    }
}

File: Java Module/Assignments in c to java/Assignment-01/Question_3.java
public class Question_3 {

    public static void main(String[] args) {
            int year=2008;
    if ((year%400==0)||(year%4==0&&year%100!=0))
    {
        System.out.printf("%d is a leap year",year);
    }
    else
    {
        System.out.printf("%d is not a leap year",year);
    }
    }
}